src/vtlengine/DataTypes/TimeHandling.py

```diff
@@ -236,9 +236,16 @@ def _meta_comparison(self, other: Any, py_op: Any) -> Optional[bool]:
             if len(other) == 0:
                 return False
             other = TimePeriodHandler(other)
-        return py_op(
-            DURATION_MAPPING[self.period_indicator], DURATION_MAPPING[other.period_indicator]
-        )
+
+        if DURATION_MAPPING[self.period_indicator] != DURATION_MAPPING[other.period_indicator]:
```

Comment:
We need to review the criteria here, I believe we should check first the duration, then the year then the period number. A ticket for the VTL community is still pending to be produced to address the criteria

Reply:
As specified in: https://github.com/VTL-Community/VTL-Community/issues/58#issuecomment-2511320746

The only way of comparing time periods is to specify left and and right time points:

2022Q1 = [2022-01-01, 2022-03-31) 2023M1 = [2023-01-01, 2023-01-31]

Now the periods could be disjoint (as in this case), so the answer to the initial question is very easy: 2023M1 is greater than 2022Q1 because in the time axis 2023M1 is completely on the right of 2022Q1.
For overlapping time periods, since usually one of the two ends coincides, the "largest" or the one with greater right end is the greater, as we have introduced a "default order" in VTL 2.1 (Y>S>Q>M) so 2022S1 >= 2022Q2 and 2022Q1>=2022M3.
I hope this can help.